src/main/java/com/rackspace/jenkins_nodepool/NodePoolDebianOpenJDKInstaller.java

```diff
@@ -27,9 +30,9 @@
     private static final String DEFAULT_OPENJDK_8_JAVA_HOME = "/usr/lib/jvm/java-8-openjdk-amd64";
 
     /**
-     * Increment this when modifying this class.
+     * Serial version UID - change this when modifying this class.
      */
-    public static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = -1021293810434797215L;
```

Comment:
My IDE flagged this one. As a best practice, the serialVersionUID is private static final, and has a value generated by the `serialver` tool in the JDK bin folder.  IntelliJ and (I think) Netbeans, can generate this value for you.  It's a hash of the class structure that should be generated (at compile-time) and used as the id (otherwise the JRE will have to generate one at run-time), which is expensive.